frontend/src/components/parties/PartyProfile.js

```diff
@@ -150,24 +167,41 @@ export class PartyProfile extends Component {
           <div className="profile__header">
             <div className="inline-flex between">
               <h1>{formatedName}</h1>
-              <AuthorizationWrapper permission={Permission.CREATE}>
-                <Button
-                  type="primary"
-                  onClick={(event) =>
-                    this.openEditPartyModal(
-                      event,
-                      parties,
-                      this.editParty,
-                      ModalContent.EDIT_PARTY(formatedName),
-                      isPerson,
-                      this.props.provinceOptions
-                    )
-                  }
-                >
-                  <img alt="pencil" className="padding-small--right" src={EDIT} />
-                  Update Party
-                </Button>
-              </AuthorizationWrapper>
+              <div>
+                <AuthorizationWrapper permission={Permission.ADMIN}>
+                  <Popconfirm
+                    placement="bottom"
+                    title={`Are you sure you want to delete the party '${formatedName}'?  Doing so will permanently
+                     remove the party and all associated roles.`}
+                    onConfirm={this.deleteParty}
+                    okText="Yes"
+                    cancelText="No"
+                  >
+                    <Button type="ghost" style={{ color: "#BC2929", borderColor: "#BC2929" }}>
```

Comment:
Okay for this button, (following up from our chat on slack)
- update the type to be "type=danger" as this is a built-in type from ant design that we haven't customized/utilized yet. 
- move all styling to themeOverride.scss, follow a similar pattern as `.ant-btn-secondary` for this new button, the class will be `.ant-btn-danger`
- For the icon, move the hex color into a constant in `constants/styles.js`

If you need further clarification let me know!

Reply:
done and pushed